cli/popper/parser.py

```diff
@@ -180,9 +180,6 @@ def validate_workflow_block(self):
             log.fail('[resolves] attribute must be a string or a list '
                      'of strings.')
 
-        if workflow_block.get('on', None):
```

Comment:
these are the actual changes that are needed in order to ignore the `on` attribute. So in other words, a workflow can still have them, but we just ignore them.